internal/pkg/cli/storage_init.go

```diff
@@ -545,18 +545,18 @@ func buildStorageInitCmd() *cobra.Command {
 	vars := initStorageVars{}
 	cmd := &cobra.Command{
 		Use:   "init",
-		Short: "Creates a new storage config file in a service's addons directory.",
-		Long: `Creates a new storage config file in a service's addons directory.
-Storage resources are deployed to your service's environments when you run
-'copilot svc deploy'. Resource names are injected into your service containers as
-environment variables for easy access.`,
+		Short: "Creates a new storage config file in a workload's addons directory.",
```

Comment:
What do you think of:
```suggestion
		Short: "Creates a new AWS CloudFormation template for a storage resource",
```